client/state/selectors/should-display-tos-update-banner.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Indicates if a Terms or Service update banner should be displayed.
+ *
+ * @param state {object}
+ * @returns {boolean} Whether the ToS update banner should be displayed.
+ */
+export const shouldDisplayTosUpdateBanner = state => state?.legal?.tos?.displayPrompt;
```

Comment:
🎉

Reply:
Having this centralized in the API has been tremendously helpful. Thanks for keeping our complexities wrapped in simplicity (I watched your talk! 😃).